### ISSUE TITLE
CHECKM on shared machine - write permission errors

### ISSUE THREAD

**Issue by wolfgangrumpf** [bxlab/metaWRAP#106]
We are running Metawrap on a multi-user cluster which is managed by an HPC team.  CheckM fails when invoked from within metawrap with this error:

```
It seems that the CheckM data folder has not been set yet or has been removed. Running: 'checkm data setRoot'.
You do not seem to have permission to edit the checkm config file
located at /opt/metawrap/1.1/envs/metawrap/lib/python2.7/site-packages/checkm/DATA_CONFIG
Please try again with updated privileges. Error was:

[Errno 13] Permission denied: '/opt/metawrap/1.1/envs/metawrap/lib/python2.7/site-packages/checkm/DATA_CONFIG'
Sorry, CheckM cannot run without a valid data folder.
```
The HPC admin team asked if you could make the following change to metawrap, which would prevent us from having to install multiple independent copies of CheckM in each user's directory:

_"Since metawrap was installed to be available to all users, the installation directory is write-protected. We had the same problem with another user (using checkm).  Can you inquire about using checkm such that the writeable files/directories can be isolated into a separate location for different users/groups?"_

In other words, if config-metawrap let me specify a local-to-me (and thus writable) directory for CheckM, it would all work!

**Comment by ursky** [comment]
As far as I understand, you cannot set the CheckM database during the run (so I don't think I can do this during the metawrap run). It needs to be set with `checkm data setRoot`. That being said, if the admin's checkm is appropriately configured to use a checkm database in a admin-accessable directory, I don't see why the user would need to modify it. In other words, I see that the users can't run `checkm data setRoot` themselves (they cant edit the config file), but if the admin configures it himself, they shouldn't have to. Did the admin try to configure it first?

**Comment by wolfgangrumpf** [comment]
I've forwarded your suggestion - I'll let you know as soon as I do!

**Comment by wolfgangrumpf** [comment]
And it's working now.  Thanks again!
